deployment/terraform/create.go

```diff
@@ -181,8 +220,34 @@ func (t *Terraform) setupAppServers(output *terraformOutput, extAgent *ssh.ExtAg
 	}
 }
 
+func (t *Terraform) setupLoadtestAgents(extAgent *ssh.ExtAgent, output *terraformOutput) error {
+	// Creating the sysadmin user.
+	// should wait the server to start
+
+	for i := 0; i < len(output.Agents.Value); i++ {
+		val := output.Agents.Value[i]
+
+		if err := t.runAgent(extAgent, val.PublicIP); err != nil {
+			mlog.Error("error while setting up an agent", mlog.String("agent name", val.Tags.Name), mlog.Err(err))
```

Comment:
Same as below, let's return the error instead of continuing. We want the entire thing to be an atomic and idempotent operation. So if it fails, the user can try it again. But when it passes, that means everything has passed perfectly.

Reply:
Fixed 👍

Reply:
I still see the continue statement :thinking: